tests/appsTest.js

```diff
@@ -9,16 +7,138 @@ fdescribe('Main: Currently testing apps management,', function () {
   var oc
 
   // TESTING CONFIGS
-  var testApp = 'someAppName'
-  var nonExistentApp = 'nonExistentApp' + timeRightNow
+  var nonExistentApp = 'nonExistentApp123'
 
   // PACT setup
   const Pact = require('@pact-foundation/pact-web')
   const provider = new Pact.PactWeb()
-  const { setGeneralInteractions } = require('./pactHelper.js')
+  const { setGeneralInteractions, ocsMeta } = require('./pactHelper.js')
+  const { validAuthHeaders, xmlResponseHeaders } = require('./pactHelper.js')
```

Comment:
You can do something like this
```suggestion
   const {
    setGeneralInteractions,
    ocsMeta,
    validAuthHeaders,
    xmlResponseHeaders
  } = require('./pactHelper.js')
```

Reply:
I prefer as it is now. I don't want to add extra lines to import the same thing.